pkgs/development/tools/xcbuild/platform.nix

```diff
@@ -0,0 +1,604 @@
+{ stdenv, sdk, writeText, platformName, xcbuild }:
+
+let
+
+  Info = {
+    CFBundleIdentifier = platformName;
+    Type = "Platform";
+  };
+
+  Version = {
+    ProjectName = "OSXPlatformSupport";
+  };
+
+  PackageTypes = [
```

Comment:
Hard to say — I think they're more part of the `.platform` than the build system that xcbuild provides. But there might be a way to have them as like templates to build off of?